README.md

```diff
@@ -27,6 +27,9 @@ This library assumes that the middlewares built on top of it act as a static
 file server. That is because static file serving is one of the most common
 use cases for HTTP/2 server push.
 
+See https://github.com/google/node-fastify-auto-push for an example. It is a
+fastify plugin for supporting auto-push.
```

Comment:
can you link fastify? `[fastify](https://www.fastify.io/)`.

Reply:
There's already a link to fastify in line 24 but sure, added.

Reply:
Oh I missed that, never mind :)